lua/lean/lean3.lua

```diff
@@ -215,7 +215,8 @@ function lean3.update_infoview(pin, bufnr, params, use_widget, opts, options)
           local clickable_event = div_event == "click" or div_event == "change"
           if clickable_event then element_div.highlightable = true end
           events[div_event] = function(this_tick, value)
```

Comment:
My bad! I should have made `value` the third parameter after I updated to be passing in the `render_fn`. Though this fixes it as well, right? So could we revert 2589253? (again, I'm a bit occupied this week so I may not be able to do much more that comment)

Reply:
(specifically, this was introduced in 27977d1)